### DISCUSSION TITLE
Plotting clocked discrete variables in OMPlot

### DISCUSSION THREAD

**Discussion by juanmigueltiscar** [OpenModelica/OpenModelica#11466] [General]
Hello, I'm not sure if this is a stupid question or not, but I can't find a way to solve it. The issue is that when I plot in OpenModelica, if I perform simulations with clocked discrete variables, the graphical representations are lines, and they can be confusing. A clocked discrete variable should not be defined where no events have occurred, so its representation should be points, not lines. Is there a way to achieve this in OpenModelica?

In other words, is there a way to create these graphs with clocked discrete variables in OpenModelica?

[Link to Modelica.Clocked.Examples.Elementary.ClockSignals](https://doc.modelica.org/Modelica%204.0.0/Resources/helpDymola/Modelica_Clocked_Examples_Elementary_ClockSignals.html#Modelica.Clocked.Examples.Elementary.ClockSignals)

Thank you.

**Comment by casella** [comment]
Known issue, see #8388. We'll try to improve this next year. Please refer to that ticket for updates.
